packages/core/src/network.js

```diff
@@ -87,6 +89,25 @@ export class Network {
     });
   }
 
+  // Validates that requestId is still valid as sometimes request gets cancelled and we have already executed
+  // _forgetRequest for the same, but we still attempt to make a call for it and it fails
+  // with Protocol error (Fetch.failRequest): Invalid InterceptionId.
+  async send(session, method, params) {
+    /* istanbul ignore else: currently all send have requestId */
+    if (params.requestId) {
```

Comment:
If all send have requestId then why are we checking this?

Reply:
There are other events that dont, and send is supposed to be used for all current and future events.